source/slang/hlsl.meta.slang

```diff
@@ -4888,7 +4888,17 @@ uint16_t asuint16(float16_t value)
 [__readNone]
 [require(cuda_glsl_hlsl_spirv, shader5_sm_5_0)]
 vector<uint16_t,N> asuint16<let N : int>(vector<float16_t,N> value)
-{ VECTOR_MAP_UNARY(uint16_t, N, asuint16, value); }
+{
+    __target_switch
+    {
+    case hlsl: __intrinsic_asm "asuint16";
+    case spirv: return spirv_asm {
+        OpBitcast $$vector<uint16_t,N> result $value
```

Comment:
```
result:$$vector<uint16_t,N> = OpBitcast %value
```

Reply:
I thought they are the same thing.
Is it more preferable style to use `result:$$XX = `?

Reply:
Yes, this is more preferred.